### PR TITLE
fix: update descriptions and examples of cmd

### PR DESCRIPTION
**Pull request by ZouRui89**
Signed-off-by: Zou Rui <21751189@zju.edu.cn>

**1.Describe what this PR did**
update descriptions and examples of cmd
**2.Does this pull request fix one issue?** 

NONE
**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

File: cli/rename.go [hunk 1]
```diff
@@ -44,11 +44,11 @@ func (rc *RenameCommand) runRename(args []string) error {
...
-Name     ID       Status    Image
-foo      71b9c1   Running   docker.io/library/busybox:latest
+Name     ID       Status    Image                              Runtime
+foo      71b9c1   Running   docker.io/library/busybox:latest   runc
 $ pouch rename foo newName
 $ pouch ps
-Name     ID       Status    Image
-newName  71b9c1   Running   docker.io/library/busybox:latest
+Name     ID       Status    Image                              Runtime
+newName      71b9c1   Running   docker.io/library/busybox:latest   runc
```
Comment (allencloud) [inline cli/rename.go]: Please check why this command does not align in the right place.

File: cli/version.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 
 // versionDescription is used to describe version command in detail and auto generate command doc.
 // TODO: add description
-var versionDescription = ""
+var versionDescription = "Display the information of pouch version"
```
Comment (allencloud) [inline cli/version.go]: We can add more details about client side and daemon side. And what kind of version will be displayed in output.

File: cli/rename.go (hunk 1 above)
Comment (ZouRui89) [reply cli/rename.go]: Clipboard error. Everything goes fine with rename cmd.

File: cli/version.go (hunk 2 above)
Comment (ZouRui89) [reply cli/version.go]: But based on the example "
GoVersion:       go1.9.1
 KernelVersion:
 Os:              linux
 Version:         0.1.0-dev
 APIVersion:      1.24
 Arch:            amd64
 BuildTime:       2017-12-18T07:48:56.348129663Z
 GitCommit: ",
it seems that the information is just about one single side, not about client side and daemon side. 
In addition, you think that we should list detailed versions like GoVersion and KernelVersion?

File: cli/version.go [hunk 3]
```diff
@@ -49,7 +49,15 @@ func (v *VersionCommand) runVersion() error {
 }
 
 // versionExample shows examples in version command, and is used in auto-generated cli docs.
-// TODO: add example
 func versionExample() string {
-	return ""
+	return `$ pouch version
+GoVersion:       go1.9.1
+KernelVersion:
```
Comment (allencloud) [inline cli/version.go]: Could you add a follow-up PR to make daemon side return KernelVersion?

**Comment by allencloud** [comment]
LGTM

File: cli/version.go (hunk 3 above)
Comment (ZouRui89) [reply cli/version.go]: Well..I have spent some time on this. But I have not found the available library function to call.   Now the only way I can figure out is to open /proc/version on the user' computer and get the kernel information. I am not sure whether this is acceptable or not due to user privacy.

Comment (allencloud) [reply cli/version.go]: I am afraid you should design a utils packages in ./pkg to make this.

**Pull request merged**